src/main/scala/strawman/collection/mutable/Set.scala

```diff
@@ -69,9 +69,7 @@ trait Set[A]
   }
 }
 
-object Set {
-  def apply[A](xs: A*): Set[A] = ???
-}
+object Set extends IterableFactory.Delegate[Set](HashSet)
```

Comment:
I'm still unsure how much we should expose default implementations. We probably want one for `Set` because `SortedSet` has to fall back to building something (although this could be up to the individual SortedSet implementation) but do we want to expose this as a factory so that users can call `.to(Set)`? Or should they have to choose what type they really want? It's probably less controversial here than when it comes to `.to(Seq)`. Should this build a List, a Vector, an ImmutableArray or something else?